terraform/services/config/values/test.sopsw.yaml

```diff
@@ -3,7 +3,7 @@
 /cdap/sensitive/account/security_events_slack_webhook_url: ENC[AES256_GCM,data:aiW0cRzw2ZBM+WmgyXYeEGTIgP3Rm2vL6wa8+mGYyMnMQ718iODap66yNHLkOttq2Lz3V+HDdvjwYXDfG+IBvj9zVpQTjC3EabvByOCIeA==,iv:3GRLuWVe8Z5N3I9JY0DVyrPcwmIv+5vMwIS5ZR4P/v0=,tag:N4Hq6cI/JUUkCzR0BaL/sA==,type:str]
 /cdap/sensitive/bucket-access-logs-bucket: ENC[AES256_GCM,data:a9oeAjfhElMm1bceKXkQiZuCnvKGJ4KZdUDHHgbcJhV/KDOJ/Kms+ThAl4s1,iv:86Bff+m1IP0eMbEEzTql64XjeXqkXqEpWXHc9HElkAU=,tag:bAIb8jWb4VQXw603f3zv1g==,type:str]
```

Comment:
The bucket-access-logs-bucket param should be managed in the bucket-access-logs service, not by SOPS. It isn't sensitive. It's just a pointer to the bucket created by that service.

Drop this param here and in prod.sopsw.yaml. And drop all changes to bucket-access-logs/main.tf.